giskard/visualization/templates/scan_report/markdown/huggingface.md

```diff
@@ -1,12 +1,33 @@
+{% if issues|length == 0 %}
```

Comment:
I think we can just put this in cicd:

https://github.com/Giskard-AI/cicd/blob/main/giskard_cicd/automation/post_discussion.py#L15

Reply:
Removed and added in cicd